source/public/index.html

```diff
@@ -6,28 +6,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="shortcut icon" href="images/favicon.ico">
 
-    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="styles/styles2.css">
```

Comment:
Could you combine the stylesheets into the original `styles.css`? That should allow for use of some repeated CSS classes (ie buttons, modal) to reduce repetition.